.storybook/main.js

```diff
@@ -3,10 +3,9 @@ const webpackConfig = require('./webpack-config');
 module.exports = {
     stories: ['../source/patterns/**/*.stories.js'],
     addons: [
-        '@storybook/addon-viewport',
-        '@whitespace/storybook-addon-html',
-        '@storybook/addon-knobs',
-        '@storybook/addon-a11y/register'
+        '@storybook/addon-essentials',
```

Comment:
[question] I assume (since you're PRing it) that none of the stuff included in Essentials conflicts with HTML Storybook?

Reply:
Right - most of these essential addons we were already using, and the new ones seem to work fine as well.